### PR TITLE
Updated the Sponsership text

### LINKED ISSUE #18: Update sponsorship text
How does this sound 

For support reach out to support@pycon.in

_Originally posted by @satyaakam in https://github.com/pythonindia/inpycon2020/pull/17_

### PR DESCRIPTION
**Pull request by Sc0rpi0n101**
Fixes #18 

CC: @ananyo2012

### CODE REVIEW

**Review by ananyo2012 — APPROVED** [review]

File: _includes/supporter.html [hunk 1]
```diff
@@ -2,14 +2,15 @@
...
+      PyCon India is a volunteer-driven event. 
+      Sponsoring the event helps bring the volunteers effort to fruition, 
+      sustain, and improve the conference and grow the community. 
+      Sponsors help us make the conference affordable, 
+      bring more facilities to our attendees, 
+      and maintain the inventory for the conference.
     </div>
     <p class="description">
-      Are you interested to support PyCon India 2020? Reach us out at 
+      Interested in supporting PyCon India 2020? Reach out to us at
```
Comment (palnabarun) [inline _includes/supporter.html]: Isn't the first question an incomplete sentence?

**Review by palnabarun — CHANGES_REQUESTED** [review]

File: _includes/supporter.html (hunk 1 above)
Comment (Sc0rpi0n101) [reply _includes/supporter.html]: Ah yes. 
How about `Reach out to us at <link> to support PyCon India 2020`?

**Review by vipulgupta2048 — APPROVED** [review]
Looks good to me after Nabarun's review is resolved.

File: _includes/supporter.html (hunk 1 above)
Comment (Sc0rpi0n101) [reply _includes/supporter.html]: ping @palnabarun

**Comment by Sc0rpi0n101** [comment]
I guess Nabarun is a little busy. Anyway, I have updated the text. 
Does anyone else have any other comments about the content?

**Review by ananyo2012 — APPROVED** [review]

File: _includes/supporter.html (hunk 1 above)
Comment (palnabarun) [reply _includes/supporter.html]: @Sc0rpi0n101 Sorry for the delay here. The last few weeks were chaotic. :sweat:

**Review by palnabarun — APPROVED** [review]

**Pull request merged**